pkgs/development/libraries/libetebase/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, rustPlatform
+, pkg-config
+, openssl
+, fetchFromGitHub
+}:
+stdenv.mkDerivation rec {
+  pname = "libetebase";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "etesync";
+    repo = "libetebase";
+    rev = "v${version}";
+    sha256 = "0r6404lgaawwkEsF3HNB+eSJ+nkEeUHUiIKUNxFDAGc=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "0w04n78b9vjsd1vaz1yrqwr3s7a38qag09qsqc71rvbsar2rmzwz";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ] ++ (with rustPlatform; [
+    cargoSetupHook
+    rust.cargo
+  ]);
+
+  buildInputs = [
+    openssl
+  ];
+
+  checkInputs = with rustPlatform; [
+    cargoCheckHook
+  ];
+
+  installFlags = [ "PREFIX=$out" ];
+
+  postInstall = ''
+    # For references in evolution-etesync
+    mkdir -p $out/include
+    ln -s $out/include/etebase/etebase.h $out/include/etebase.h
+    ln -s $out/lib/libetebase.so $out/lib/libetebase.so.0
```

Comment:
This is upstream bug, they [set the soname](https://github.com/etesync/libetebase/blob/e06fe2461249b7e2f535fc485bf9a1abd67feff9/build.rs#L13) but not the symlink. Could you open a PR upstream to do that in the [Makefile](https://github.com/etesync/libetebase/blob/master/Makefile#L42)?